### ISSUE TITLE
rename all functions with _ to proper names

### ISSUE THREAD

**Issue by morrowc** [tridentli/trident#80]
https://talks.golang.org/2014/names.slide#6

oops...

trident$ grep 'func .*_.*' src/lib/*.go | wc -l 
32
trident$ grep 'func .*_.*' src/ui/*.go | wc -l
17
pitchfork$ grep 'func .*_.*' lib/*.go | wc -l
342
pitchfork$ grep 'func .*_.*' ui/*.go | wc -l
149

and of course those will all have to be chased up and fixed in all other callers :(
I'm happy to take a stab at this, once I sort out how to actually test/compile the lot of this.

**Comment by bapril** [comment]
That's a pandoras's box that we'll get to, but we have other things to focus on, I'd make this a low priority.

**Comment by morrowc** [comment]
At Tue, 14 Feb 2017 05:29:33 -0800,
bapril <notifications@github.com> wrote:
> 
> [1  <text/plain; UTF-8 (7bit)>]
> That's a pandoras's box that we'll get to, but we have other things to focus on, I'd make this a low priority. 
> 

agreed it's a pandora's box... that's why I said I'd be happy to try
and tackle it outside of whatever things you all are up to.

**Comment by massar** [comment]
One other big reason not to is because most of those functions are not really go-style functions as in the common Go sense.

Many are CLI commands others (H_/h_) are UI commands, they are not to be referenced outside the package anyway.

**Comment by morrowc** [comment]
I don't agree that 'not to be referenced outside the package' is a valid reason to avoid code quality/readability... If I send along pull requests for renames, will they be accepted? So far the renames I've got in my local repository are pretty simple, just there are a bunch of them :)

**Comment by massar** [comment]
I am all pro code quality and readability. The format of the CLI and UI functions grew over time (before we checked that go likes it differently).

We will re-architect things when 2.0 comes around, see amongst https://github.com/tridentli/pitchfork/milestone/3

and when that comes we'll modularize/package things and we'll also take care of all the non-standard golang function names.
